sonic_data_client/non_db_client.go

```diff
@@ -494,6 +543,55 @@ func streamSample(c *NonDbClient, stop chan struct{}, sub *gnmipb.Subscription,
 	}
 }
 
+func streamOnChange(c *NonDbClient, stop chan struct{}, sub *gnmipb.Subscription) {
```

Comment:
does each subscriber have their owner  streamOnChange? If how will you handle the case two client subscribe to syslog as the same time? have you tested it?

Reply:
Do you plan to enable onchange mode for other dataset? If not you should disable them in the call flow to avoid expected issue in the server.